src/scss/modules/modals.scss

```diff
@@ -16,7 +16,7 @@
 .modal--large {
   @extend .modal;
 
-  min-width: 600px;
+  min-width: 700px;
   width: auto;
 }
```

Comment:
Just realized this will effect the form modals. A change made in https://github.com/RoundingWell/care-ops-frontend/pull/1475.

I think we should just create a new classname: `.modal--patient-search` and use that:

```css
.modal--large {
  @extend .modal;

  width: 700px;
}
```

Reply:
Okay yep makes sense. I've got the overflow ellipsis thing figured out I think. Just need to test it out some more.